default.nix

```
@@ -107,7 +109,7 @@ let ocaml_exe = name: bin: rts:
     staticpkgs.stdenv.mkDerivation {
       inherit name;
 
-      allowedRequisites = [];
+      allowedRequisites = if is_static then staticpkgs.musl else [];
```

```suggestion
      allowedRequisites = if system == "aarch64-linux" then staticpkgs.musl else [];
```
but even this doesn't help in a.t.m.